src/routes/fetchMetricsData.ts

```diff
@@ -0,0 +1,31 @@
+import { validate, Joi } from 'express-validation';
+import UserReturnData from '../UserReturnData';
+import StatusCodes from '../StatusCodes';
+
+module.exports = (app: any, dbHandler: any) => {
+  const validation = {
+    query: Joi.object({
+      competition: Joi.string().required(),
+      team_number: Joi.string().required(),
+    }),
+  }
+  app.get('/api/fetchMetricsData', validate(validation, { keyByField: true }, {}), async (req: any, res:any) => {
+    const val: UserReturnData = new UserReturnData();
+    const { competition }: Record<string, string> = req.query;
+    const teamNumber = parseInt(req.query.team_scouted, 10);
```

Comment:
inconsistent variable naming - use `team` everywhere 
```suggestion
    const team = parseInt(req.query.team, 10);
```